workers/unity/Packages/com.improbable.gdk.tools/GenerateFiles.cs

```diff
@@ -0,0 +1,38 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Tools
+{
+    public class GenerateFiles
+    {
+        private static readonly string
+            ResourcesDir = Path.GetFullPath(Path.Combine(Application.dataPath, "Resources"));
+
+        [MenuItem("SpatialOS/Generate Dev Authentication Token", false, MenuPriorities.GenerateDevAuthToken)]
+        private static void GenerateDevAuthToken()
+        {
+            var receivedMessage = string.Empty;
+            RedirectedProcess
+                .Command(Common.SpatialBinary)
+                .WithArgs("project", "auth", "dev-auth-token", "create", "--description", "Dev Auth Token")
```

Comment:
Should the last one be "\"Dev Auth Token\"" so that the process doesn't perceive space as new argument?